### DISCUSSION TITLE
Welcome to EasyLauncher Discussions!

### DISCUSSION THREAD

**Discussion by CreativeCodeCat** [DroidWorksStudio/EasyLauncher#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Antlersandsunbeams** [comment]
Heya, I'm a big fan of mLauncher, so thanks for all your work on it! I'm excited to see what you do with ezL.

I've got no real coding experience, but I'd like to start learning Kotlin; is there a useful way to contribute to your projects that isn't strictly code?

**Reply by CreativeCodeCat** [reply]
If you feel there is anything missing feel free to join telegram and let us know and I'm sure we can find something you could help with.

**Reply by due3die** [reply]
> If you feel there is anything missing feel free to join telegram and let us know and I'm sure we can find something you could help with.

What is the telegram link? So that I can join.

I need to give a bug report for nightly builds and regular builds.

My Telegram ID: @SUDHARSAN_M

**Reply by CreativeCodeCat** [reply]
> What is the telegram link?

https://t.me/DroidWorksStudio it's also on the readme page

**Comment by ddogfoodd** [comment]
Hey, whats the difference between the Internet and non-internet release?

**Reply by CreativeCodeCat** [reply]
One has INTERNET access one does not. The one with needs for weather widget which is disabled in the no internet build.

**Reply by ddogfoodd** [reply]
Ah I see, thanks! Am new to Android and would have thought only the weather app providing the widget data would need internet access. Then again I guess it might be a weather widget provided by the launcher. Will try both.

**Reply by CreativeCodeCat** [reply]
> Ah I see, thanks! Am new to Android and would have thought only the weather app providing the widget data would need internet access. Then again I guess it might be a weather widget provided by the launcher. Will try both.

All the widgets on the launcher are created by the launcher
